frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -2,8 +2,21 @@ import React from 'react';
 import './GameStatistics.css';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { Button } from '../../Button/Button.js';
+import { getStatistics } from '../../../apis/getStatistics.js';
+import { useEffect, useState } from 'react';
+import { TeamID } from '../../../constants/score.js';
 
 function GameStatistics({ finalScores, onNewGameRequested }) {
+  const [data, setData] = useState({});
+  useEffect(() => {
+    getStatistics().then((result) => {
+      if (result?.error) alert(result.error);
+      setData(result.data);
```

Comment:
You're checking the result against possible null, but only inside the if statement. `null` and `undefined` are falsy values, so it does not resolve to `true` here (alert will not be called).

Another thing is that if a callback function provided in `.then` is called it means arguments are always provided, so `result` never be undefined or null until you return it in that way in `getStatistics`.